doc/samples/samples_list.rst

```diff
@@ -27,6 +27,13 @@ The following table demonstrates the variants' configuration for the supported d
 
 `*` The `nRF54L15 PDK`_ does not support LoRa and FSK configurations.
 
+.. note::
+
+   Targets `nRF5340 DK`_  and `Thingy53`_  use `MCUBOOT_MODE_OVERWRITE_ONLY` configuration for bootloader. This means, that after the DFU image swap, there is no rollback mechanism in case new image is not functional.
+   This limitation is due to split between app and net core. If the default MCUBOOT mode to be set, the rollback would only apply to the app core, and not to the net core. This could lead to missmatch between app and net core.
+   Please verify the image to be updated before performing DFU on large scale as faulty image may lead to unrecoverable state of the device. 
+   Other platforms use default MCUBOOT mode (`MCUBOOT_MODE_SWAP_WITHOUT_SCRATCH`), and it requires new image to be confirmed (either externally, or by the new application itself), otherwise on next reboot mcuboot will revert the change. 

```

Comment:
```suggestion
   The `nRF5340 DK`_  and `Thingy53`_  targets use the ``MCUBOOT_MODE_OVERWRITE_ONLY`` bootloader configuration, which does not support rollback. 
   This means that in case the swapped DFU image fails, it cannot be reverted to the previous version.
   This limitation is due to the split between the app and net core. 
   Setting the default MCUboot mode would allow rollback only for the app core, not the net core, potentially leading to mismatches between the two.
   Before performing DFU on large scale, you must verify the update image, as a faulty image could make the device unrecoverable. 
   Other platforms use the default MCUboot mode (``MCUBOOT_MODE_SWAP_WITHOUT_SCRATCH``), which requires new image verification (either externally or by the new application)  to prevent MCUboot from reverting the changes on the next reboot. 
```